pkgs/by-name/go/google-chrome/package.nix

```diff
@@ -142,6 +142,17 @@ in stdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  passthru = {
+    updateScript = writeScript "update-google-chrome.sh" ''
+      #!/usr/bin/env nix-shell
+      #!nix-shell -i bash -p curl jq common-updater-scripts
+      url="https://versionhistory.googleapis.com/v1/chrome/platforms/linux/channels/stable/versions/all/releases"
+      response=$(curl --silent $url)
```

Comment:
Is there a particular reason why you want to quote a command substitution in a variable assignment in bash?

For consistency?

Genuinely curious.

Reply:
Okay so I've always done this. Your comment made me read some more, and indeed it seems like `FOO=$(bar)` is functionally equivalent to `FOO="$(bar)"`, but indeed for consistency I'm in favour. Not blocking, though.

Reply:
I agree with Jon that we should quote these variables, else bash might not consider them as a single string value and split the words into an array.

And we are doing that in line 151 and 152.

https://github.com/NixOS/nixpkgs/blob/d95e9bf3a61395f00b18379d9fb6388d3e6645f9/pkgs/by-name/go/google-chrome/package.nix#L152

Double quoting them again is not a good idea IMO, although, we can double quote only during variable assignment and then just use `$version` when we need it instead of again wrapping like `"$version"`

Reply:
Even more important ``set -eou pipefail`` is missing and the curl is not told to fail with --fail, so it could be that the script ends up with an empty string as version.